src/toast/tests/_helpers.py

```diff
@@ -83,6 +83,17 @@ def create_comm(mpicomm):
     return toastcomm
 
 
+def close_data(data):
```

Comment:
The destructor of the `Data` class does not destroy / close the `toast.Comm` which is passed at construction time (i.e. it does not trigger the `Free()` method of the MPI communicators).  This is intentional, since that `Comm` object might be used in other data objects in a normal workflow.  We have unit test helper functions that create a `Comm` and a `Data` instance and return the data object.  To prevent weird MPI things in the case of the unit tests (where hundreds of `Comm` objects are being created), I found it was necessary to cleanly destroy the `Data` **and** `Comm` classes at the end of each unit test.  So `close_data(data)` is equivalent to `del data.comm; del data`.  Inside this helper function I go one step further and explicitly call `data.comm.close()` before the destructor, since python destructors can sometimes do unexpected things in the order of deletion of instance attributes.

So- this function is doing more than just the normal destructor of the input data.  Maybe I should rename it to `close_data_and_comm()`?  Something else?